apps/Cargo.toml

```diff
@@ -44,9 +44,25 @@ dev = ["anoma/dev"]
 std = ["ed25519-dalek/std", "rand/std", "rand_core/std"]
 # for integration tests and test utilies
 testing = ["dev"]
+ABCI = [
```

Comment:
I think we should add this one to default features for now

Reply:
I tried that but ran into a lot of trouble when trying to enable the ABCI++ feature. Both would get turned on

Reply:
Unfortunately, this doesn't work nicely with cargo workspace :(